models/signups.js

```diff
@@ -5,7 +5,8 @@ const Signups = new Schema({
 	userName: String,
 	ip: String,
 	type: String,
-	email: String
+	email: String,
+	unobfuscatedIP: String
```

Comment:
Why can't you use `ip` from line 6?

Reply:
The way that it currently works, the `ip` field of signups stores only the obfuscated IP, which makes it often hard to link signups to other accounts after resets etc. This `unobfuscatedIP` field is only there for reference if ever needed directly in the DB to compare to other accounts after a reset when the obfuscated IP becomes irrelevant. (The other solution could be to obfuscate IPs on the fly in the signups page, but that would probably cause similar lag as in modlog so I figured this is probably best)

Reply:
Ah we should probably rename the `ip` field idk